tests/PhpPact/Consumer/Model/ConsumerRequestTest.php

```diff
@@ -49,7 +49,7 @@ public function testSerializingWhenPathUsingMatcher()
         $this->assertEquals('PATCH', $model->getMethod());
         $this->assertEquals(['Content-Type' => ['application/json']], $model->getHeaders());
         $this->assertEquals(['food' => ['milk']], $model->getQuery());
-        $this->assertEquals('{"value":"\/somepath\/474d610b-c6e3-45bd-9f70-529e7ad21df0\/status","regex":"\\\\\\/somepath\\\\\\/[0-9a-f]{8}(-[0-9a-f]{4}){3}-[0-9a-f]{12}\\\\\\/status","pact:matcher:type":"regex"}', $model->getPath());
+        $this->assertEquals('{"pact:matcher:type":"regex","regex":"\\\\\\/somepath\\\\\\/[0-9a-f]{8}(-[0-9a-f]{4}){3}-[0-9a-f]{12}\\\\\\/status","value":"\/somepath\/474d610b-c6e3-45bd-9f70-529e7ad21df0\/status"}', $model->getPath());
```

Comment:
This is a real pity that order could not be preserved, or the model could not be JSON serialised to show that only one thing has really changed.

Reply:
Reverted the order and then use better assert method `assertJsonStringEqualsJsonString`

Reply:
I don't think the order is important for Rust to work.

I can just change this [line](https://github.com/pact-foundation/pact-php/blob/ffi/src/PhpPact/Consumer/Matcher/Matchers/GeneratorAwareMatcher.php#L42), but I think there are many places to be updated. So I'm just lazy.